README.md

```diff
@@ -1,90 +1,39 @@
-# Serverless SaaS - Reference Solution
+# aws-saas-factory-ref-solution-serverless-saas
 
-**[Feedback & Feature request](https://www.pulse.aws/survey/EHE3TICQ)** | **[Documentation](DOCUMENTATION.md)**
```

Comment:
why did we remove the documentation reference?

Reply:
Added links back in.